website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,189 @@
+
+**Placeholder text**
+
+Placeholder text can be added to the trigger to provide the user additional context. However, this information should not be required for them to carry out a task.
+
+![Showing placeholder text inside of the trigger, still indicating no selection is made, but with some additional guidance](/assets/components/form/super-select/super-select-trigger-placeholder-text.png)
+
+**Filled**
+
+When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed.
+
+![Showing preselected options in both a single and multiple variant. This helps show something has been preselected previously and the user can update their options](/assets/components/form/super-select/super-select-trigger-filled.png)
+
+## Before and after options
+
+The before and after options are areas above and below the list, independent from the list of options. Predefined components (like search) or custom content can occupy these spaces.
+
+### BeforeOptions
+
+BeforeOptions allows for either a search field or custom content to be displayed before the options list.
+
+!!! Warning
+
+Currently, the Multiple component includes search within an area of the trigger instead of displaying it in the beforeOptions. This causes a nested interaction which is an accessibility violation. We are in the process of working with the Ember Power Select maintainers to resolve this issue and make it the same as in the Single component.
+
+!!!
+
+#### Search
+
+If enabled, search allows the user to filter results based on a query.
+
+![Search enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-before-options-search.png =836x*)
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Custom content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-before-options-custom-content.png =836x*)
+
+!!! Dont
+
+We recommend against placing heavy UI elements in custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing such content outside the list relative to the trigger for better visibility.
+
+![Custom content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-dont-visually-heavy.png =836x*)
+
+!!!
+
+### AfterOptions
+
+AfterOptions allows for either an HDS standardized footer or custom content to be displayed after the options list. The footer has additional functionality in the Multiple variant than the Single.
+
+#### Footer
+
+By default, the footer displays below the options list and contains content and/or buttons which provide additional context or functions related to the options.
+
+##### Single
+
+The footer for the Single component includes a total count, which is customizable to fit the context.
```

Comment:
Here you use "Single component" but elsewhere you use "Single variant", would probably be good to align on a single consistent term. I'll try to catch more of these as a go through

Reply:
It should be single component. I thought I'd caught this but ill do another scrub through.